apps/cli/src/transforms/include.ts

```diff
@@ -0,0 +1,26 @@
+import fs from 'fs';
+import type { GenericNode } from 'mystjs';
+import { parseMyst, Root } from '../myst';
+import { selectAll } from 'mystjs';
+import { join, dirname } from 'path';
+import { ISession } from '../session';
+
+/**
+ * This is the {include} directive, that loads from disk.
+ *
+ * RST documentation:
+ *  - https://docutils.sourceforge.io/docs/ref/rst/directives.html#including-an-external-document-fragment
+ */
+export function includeFilesDirective(session: ISession, filename: string, mdast: Root) {
+  const includeNodes = selectAll('include', mdast) as GenericNode[];
+  const dir = dirname(filename);
+  includeNodes.forEach((node) => {
+    const file = join(dir, node.file);
+    if (!fs.existsSync(file)) {
+      session.log.error(`Include Directive: Could not find "${file}" in "${filename}"`);
+    }
+    const content = fs.readFileSync(file).toString();
+    const children = parseMyst(content).children as GenericNode[];
```

Comment:
What about non md included files - not supported (do we validate?)? included as raw text?

Reply:
Yeah, good point. There are some more options on the import from RST that we could support as well.

Reply:
https://docutils.sourceforge.io/docs/ref/rst/directives.html#including-an-external-document-fragment

Reply:
I am going to leave this to the future. :)